projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -0,0 +1,1067 @@
+
+> By default, the search API will only return results that have audienceType set as everyone. In order to search for a specific audienceType you'll need to disable the defaultfilter for audienceType. In the [default filters guide](default-filters.md) you can find how you can reset this filter.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers`
+
+**Possible values**
+
+`everyone` `members` `education`
+
+**Example**
+
+```
+GET /events/q=audienceType:members&audienceType=*
+```
+
+### availableRange
+
+Using the `availableRange` field, you can get all events and places that were available in a given range in the past, or will be available in a given range in the future.
+
+> By default, the search API will only return results that are currently available. In order to also retrieve results that are not available (yet), you'll need to disable the default filters for `availability`. You can reset this default as described in the [default filters guide](default-filters.md).
+
+Most events in UiTdatabank have a limited availability, from the time they are published (or their scheduled publication date has been reached) until the end date of the event. Specific types of events are only available until the start date of an event (e.g. a course series).
+
+Places are considered to be permanently available, starting when they are published (or, again, when their scheduled publication date has been reached). A small portion of the events is permanent as well, depending on their calendar information.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers`
+
+**Possible values**
+
+Any dateFrom and dateTo date in ISO-8601.
+
+**Example**
+
+```
+GET /events/?q=availableRange:[2023-01-01T00\:00\:00%2B01\:00 TO 2023-03-31T23\:59\:59%2B01\:00]&availableFrom=*&availableTo=*
+```
+
+### bookingAvailability
+
+The `bookingAvailability` indicates whether there are still tickets or seats left for an event or not.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers`
+
+**Possible values**
+
+`available` `unavailable`
+
+**Example**
+
+Get all events that are not sold out or fully booked:
+
+```
+GET /events/?q=bookingAvailability:available
+```
+
+Get all places that are permanently closed:
+
+```
+GET /places/?q=bookingAvailability:unavailable
+```
+
+TO DO: link to booking availability guide when that guide is finished.
+
+### calendarType
+
+Every event and place in UiTdatabank has one of the following four calendarTypes:
+
+* `Single`: the event occurs on a single date, indicated by a single startDate and endDate.
+* `Multiple`: the event occurs on multiple dates, and has multiple subEvent entries with each a different startDate and endDate.
+* `Periodic`: the event or place runs for a specific period as indicated by its startDate and endDate, and can optionally have openingHours.
+* `Permanent`: the event or place is permanent and has no startDate or endDate, but it can optionally have openingHours.
```

Comment:
(Copied from older comment on previous filename)

I think it's best to not capitalize the types here, as they are always lowercase in the APIs

Reply:
Fixed in fd60622e4f3630aa90380419d0ae92f5486c6611 I see 👍